### DISCUSSION TITLE
Can CSS size be reduced ?

### DISCUSSION THREAD

**Discussion by zwoop** [razonyang/hugo-theme-bootstrap#274] [Q&A] [answered]
I've disabled Math (KaTeX) and the diagram features, but the CSS generated are still pretty darn big. In my case, they end up being (uncompressed) 144,718 bytes. Google PageSpeed complains about a significant portion of it being unused. Is there something else I should disable to reduce this size?

My current page speed score is only 82 / 86 respectively, when I played with the "Clarity" theme, it scored 100/100. Not complaining about this theme, I find it much nicer in look and feel, but hoping I can optimize it a bit more :).

**Comment by razonyang** [comment]
I'll focus on the performance after implement PWA #263.

**Comment by razonyang** [comment, marked as answer]
Fixed by #276. I also recommend enabling compression on your server side. For example, the transferred CSS size of demo site is only 16.1KB.

**Comment by zwoop** [comment]
Yep, this moves my site to 100/100 as well! And yes, compression was already on :).

Thanks for the quick update.
